File: ProjetoProgSofApp/php/src/model/PacoteDeTurismo.php
<?php

namespace model;

use Doctrine\ORM\Mapping as ORM;

#[ORM\Entity]
#[ORM\Table(name: 'tb_pacote_turismo')]
class PacoteDeTurismo extends GenericModel
{
    #[ORM\Column(type: 'string', name: 'nome_pacote', nullable: true)]
    private $nome;

    #[ORM\Column(type: 'decimal', name: 'preco_pacote', precision: 10, scale: 2, nullable: true)]
    private $preco;

    #[ORM\ManyToOne(targetEntity: Destino::class, fetch: 'EAGER')]
    #[ORM\JoinColumn(name: 'destino_id', nullable: true)]
    private $destino;

    public function getNome()
    {
        return $this->nome;
    }

    public function setNome($nome): void
    {
        $this->nome = $nome;
    }

    public function getPreco()
    {
        return $this->preco;
    }

    public function setPreco($preco): void
    {
        $this->preco = $preco;
    }

    public function getDestino()
    {
        return $this->destino;
    }

    public function setDestino($destino): void
    {
        $this->destino = $destino;
    }
}

File: src/main/java/br/edu/ifpr/bsi/ProjetoProgSofApp/models/PacoteDeTurismo.java
package br.edu.ifpr.bsi.ProjetoProgSofApp.models;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.FetchType;
import jakarta.persistence.JoinColumn;
import jakarta.persistence.ManyToOne;
import jakarta.persistence.Table;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Entity
@Table(name = "tb_pacote_turismo")
public class PacoteDeTurismo extends GenericModel {

    @Column(name = "nome_pacote")
    private String nome;

    @Column(name = "preco_pacote")
    private Double preco;

    @ManyToOne(fetch = FetchType.EAGER)
    @JoinColumn(name = "destino_id")
    private Destino destino;
}